lib/WasmReader/WasmBinaryReader.cpp

```diff
@@ -438,33 +438,38 @@ const uint32 WasmBinaryReader::EstimateCurrentFunctionBytecodeSize() const
     return m_funcState.size;
 }
 
+WasmOp WasmBinaryReader::ReadPrefixedOpCode(WasmOp prefix, bool isSupported, const char16* notSupportedMsg)
+{
+    CompileAssert(sizeof(WasmOp) >= 2);
+    if (!isSupported)
+    {
+        ThrowDecodingError(notSupportedMsg);
+    }
+    CheckBytesLeft(1);
+    ++m_funcState.count;
```

Comment:
Right now we don't need to read more than 1 byte past the prefix.
However, I cannot find in the spec nor any proposal if the opcode following the prefix is a byte or a LEB128.
Anyone has more context on this ?

Reply:
I think the post prefix opcode should remain to be one byte. Trying to confirm this from the spec/discussions.

The design choice of adding the prefix was to restrict the mvp opcodes to a single byte and allow ample space to add post mvp features with significant number of opcodes (like simd). This gave enough space for all post mvp features and cant think of a need for LEB128 opcode encoding (most of the features with prefix are adding much less than 256 new opcodes).

Reply:
Thanks for the research @arunetm 
Looks like we are doing the right thing regarding prefixes